### PR TITLE
Rename experimental to preview

### PR DESCRIPTION
**Pull request by RichardIrons-neo4j**

### CODE REVIEW

File: Neo4j.Driver/Neo4j.Driver/Preview/ExperimentalExtensions.cs [hunk 1]
```diff
@@ -16,10 +16,10 @@
...
 using System.Collections.Generic;
-using Neo4j.Driver.Experimental.FluentQueries;
 using Neo4j.Driver.Internal;
+using Neo4j.Driver.Preview.FluentQueries;
 
-namespace Neo4j.Driver.Experimental;
+namespace Neo4j.Driver.Preview;
 
 /// <summary>
 /// There is no guarantee that anything in Neo4j.Driver.Experimental namespace will be in a next minor version.
```
Comment (thelonelyvulpes) [inline Neo4j.Driver/Neo4j.Driver/Preview/ExperimentalExtensions.cs]: `Neo4j.Driver.Experimental` exists in comments might be worth do a sed s/Neo4j.Driver.Experimental/Neo4j.Driver.Preview/g

Comment (thelonelyvulpes) [reply Neo4j.Driver/Neo4j.Driver/Preview/ExperimentalExtensions.cs]: Should we also rename this class?

Comment (RichardIrons-neo4j) [reply Neo4j.Driver/Neo4j.Driver/Preview/ExperimentalExtensions.cs]: doy, will do

**Review by thelonelyvulpes — APPROVED** [review]
~Experimental~Preview

**Pull request merged**